frontend/src/scenes/PreflightCheck/index.tsx

```diff
@@ -150,26 +153,46 @@ export function PreflightCheck(): JSX.Element {
 
     return (
         <div style={{ minHeight: '100vh' }}>
-            <Space direction="vertical" className="space-top" style={{ width: '100%', paddingLeft: 32 }}>
-                <PageHeader title="Welcome to PostHog!" caption="Understand your users. Build a better product." />
-            </Space>
-            <Col xs={24} style={{ margin: '0 16px' }}>
-                <h2 className="subtitle text-center space-top">
-                    We're glad to have you here! Let's get you started with PostHog.
-                </h2>
-            </Col>
+            <Row
+                style={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    paddingTop: 32,
+                    paddingBottom: 32,
+                    backgroundColor: '#eeefe9',
+                }}
+            >
+                <img src={posthogLogo} style={{ width: 157, height: 30 }} />
+            </Row>
+            <Row style={{ display: 'flex', justifyContent: 'center', paddingBottom: 16 }}>
+                <PageHeader title="Lets get started..." />
+            </Row>
+            <Row style={{ display: 'flex', justifyContent: 'center' }}>
+                <div style={{ width: 960 }}>
+                    <Steps current={0}>
+                        <Step title="Preflight check" subTitle="1 min" description="Prepare your instance" />
+                        <Step
+                            title="Event capture"
+                            subTitle="15 mins"
```

Comment:
What about 10? 😉

Reply:
I guess this is mostly out of our control - depends how difficult their app is to instrument. I felt 15mins was long enough to instrument something on a more complex [maybe mobile] app (so we don't make it sound easier than it is) but short enough not to put people off forging ahead.